### PR TITLE
RC_Channel: fix typo

### PR DESCRIPTION
**Pull request by mirkix**

### CODE REVIEW

**Review by khancyr — APPROVED** [review]
LGTM, NFC

File: libraries/RC_Channel/RC_Channel.cpp [hunk 1]
```diff
@@ -896,7 +896,7 @@ bool RC_Channel::run_aux_function(aux_func_t ch_option, AuxSwitchPos pos, AuxFun
     const bool ret = do_aux_function(ch_option, pos);
 
     // @LoggerMessage: AUXF
-    // @Description: Auixillary function invocation information
+    // @Description: Auxillary function invocation information
```
Comment (Williangalvani) [inline libraries/RC_Channel/RC_Channel.cpp:899]: should this be "Auxiliary"? Looks like you got rid of the wrong letter
```suggestion
    // @Description: Auxiliary function invocation information
```

Comment (magicrub) [reply libraries/RC_Channel/RC_Channel.cpp:899]: yup, it's a single L

**Comment by magicrub** [comment]
@peterbarker Did you not want the correct spelling? New PR here.. https://github.com/ArduPilot/ardupilot/pull/19686

**Pull request merged**